docs/user-guide/platform-linux.md

```diff
@@ -69,7 +69,6 @@ trizen -S pegasus-frontend-git
 yaourt -S pegasus-frontend-git
 # etc.
 ```
-
```

Comment:
I think you've removed a blank line here by accident.

Reply:
Yup, saw it after I submitted - no idea how